llvm/lib/Transforms/Scalar/StructurizeCFG.cpp

```diff
@@ -85,7 +86,43 @@ using PhiMap = MapVector<PHINode *, BBValueVector>;
 using BB2BBVecMap = MapVector<BasicBlock *, BBVector>;
 
 using BBPhiMap = DenseMap<BasicBlock *, PhiMap>;
-using BBPredicates = DenseMap<BasicBlock *, Value *>;
+
+using MaybeCondBranchWeights = std::optional<class CondBranchWeights>;
+
+class CondBranchWeights {
```

Comment:
Kind of weird this would be defined here, maybe later should just move it to ProfDataUtils (which could use some modernization)

Reply:
Thanks ! I'll do that in a separate PR.